openhands/events/common.py

```diff
@@ -0,0 +1,6 @@
+from enum import Enum
```

Comment:
Let's move the `FileEditSource` to `openhands/events/action/files.py`, remove the file `openhands/events/common.py` and fix all the imports.

Reply:
I think it's also used in observation as well

Reply:
Then how about we put it in openhands/events/event.py